core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.css

```diff
@@ -530,7 +530,7 @@
     white-space: nowrap;
     width: 100%;
   }
-  @media screen and (max-width: 600px) {
+  @media screen and (min-width: 769px) {
```

Comment:
Order these media queries in descending order of max-widths so that we don't need to specify the min-widths.